cura/CuraApplication.py

```diff
@@ -1612,10 +1615,25 @@ def readLocalFile(self, file, skip_project_file_check = False):
                 self.deleteAll()
                 break
 
-        if not skip_project_file_check and self.checkIsValidProjectFile(file):
+        is_project_file = self.checkIsValidProjectFile(file)
+        project_mode = force_project_mode
```

Comment:
Useless alias. You can use `force_project_mode` for the rest of the code or rename the parameter to `project_mode`. I'd prefer `force_project mode` (your new name) since it's a bit more explicit.

Reply:
Okay, I'm used to languages where mutating parameters is considered bad style.
I'm going with `project_mode` though